install/boot_vms.yml

```diff
@@ -0,0 +1,108 @@
+---
+
+# Playbook to create client and server vm on Openstack
```

Comment:
Can this be factored out into its own PR as it does not appear to be tightly related to pbench agent installation?

Reply:
This PR supports Pbench agent installation on Openstack Vm's, Computes, and Trafficgen host if enabled. This playbook creates the Uperf client and server vm.

Reply:
Sure, but this playbook appears to be completely independent of what it is being used for, which makes it verifiable on its own and not tied to anything pbench.